scripts/flash/dispute-contribute.js

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var chalk = require("chalk");
+var getTime = require("./get-timestamp");
+const { getPrivateKeyFromString } = require("../dp/lib/get-private-key");
+const repFaucet = require("../rep-faucet");
+var BigNumber = require("bignumber.js");
+var displayTime = require("./display-time");
+var setTimestamp = require("./set-timestamp");
+
+const day = 108000; // day
+
+function disputeContributeInternal(augur, marketID, outcome, amount, disputerAuth, invalid, auth, callback) {
+  repFaucet(augur, disputerAuth, function (err) {
+    if (err) { console.log(chalk.red("Error"), chalk.red(err)); callback(err); }
+    if (err) return console.error(err);
+    if (!invalid) { invalid = false; } else { invalid = true; }
+    augur.markets.getMarketsInfo({ marketIDs: [marketID] }, function (err, marketsInfo) {
+      var market = marketsInfo[0];
+      var marketPayload = { tx: { to: marketID } };
+      augur.api.Market.getFeeWindow(marketPayload, function (err, feeWindowId) {
+        var feeWindowPayload = { tx: { to: feeWindowId } };
+        augur.api.FeeWindow.getStartTime(feeWindowPayload, function (err, feeWindowStartTime) {
```

Comment:
Error needs to be handled.  (This is true for all of these functions, although I'm not going to add a comment for every one.)

Reply:
added error handling, tried to find them all in all scripts.